### PR TITLE
Feat/size guide

### PR DESCRIPTION
**Pull request by zweroboy1**
## Describe your changes

- Added Sizes Guide page.
- Added new eslint rules, fixed errors.
- Add files for deploy site on Netlify.

## Issue ticket code (and/or) and link

- [Link to JIRA ticket](https://2024internship.atlassian.net/browse/SCRUM-62)

### **General**

- [x] Assigned myself to the PR
- [x] Assigned the appropriate labels to the PR
- [x] Assigned the appropriate reviewers to the PR
- [ ] Updated the documentation
- [x] Performed a self-review of my code
- [x] Types for input and output parameters
- [x] Don't have "any" on my code
- [x] Used the try/catch pattern for error handling
- [x] Don't have magic numbers
- [x] Compare only with constants not with strings
- [x] No ternary operator inside the ternary operator
- [x] Don't have commented code
- [x] no links in the code, env links should be in env file (for example: server url), constant links (for example default avatar URL) should be in constant file.
- [x] Used camelCase for variables and functions
- [x] Date and time formats are on the constants
- [x] Functions are public only if it's used outside the class
- [ ] No hardcoded values
- [ ] covered by tests
- [x] Check your commit messages meet the [conventional commit format](https://www.conventionalcommits.org/en/v1.0.0/).

### Frontend

- [x] Components and business logic are separated
- [x] Colors, Font Size, and Font Name is on the theme or in the constants
- [x] No text in the components, use i18n approach
- [x] No inline styles
- [x] Imports are absolute
- [x] Attach a screenshot if PR has visual changes.

image
image

### CODE REVIEW

File: src/components/Footer/index.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { useTranslation } from 'react-i18next';
+
+function Footer() {
+  const { t } = useTranslation();
+
+  return <div style={{ height: '68px' }}>{t('header.home')}</div>;
```
Comment (lyudad) [inline src/components/Footer/index.tsx]: 1.3.4

File: src/components/Header/index.tsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { useTranslation } from 'react-i18next';
+
+function Header() {
+  const { t } = useTranslation();
+
+  return <div style={{ height: '68px' }}>{t('header.home')}</div>;
```
Comment (lyudad) [inline src/components/Header/index.tsx]: 1.3.4

File: src/components/Footer/index.tsx (hunk 1 above)
Comment (zweroboy1) [reply src/components/Footer/index.tsx]: fixed

File: src/components/Header/index.tsx (hunk 2 above)
Comment (zweroboy1) [reply src/components/Header/index.tsx]: fixed

**Pull request merged**